Sources/Response.swift

```diff
@@ -3,13 +3,16 @@ public struct Response: Message {
     public var status: Status
     public var headers: Headers
     public var body: Body
-    public var storage: [String: Any] = [:]
+    public var cookies: Set<Cookie>
+    public var storage: [String: Any]
 
-    public init(version: Version, status: Status, headers: Headers, body: Body) {
+    public init(version: Version, status: Status, headers: Headers, body: Body, cookies: Set<Cookie>) {
         self.version = version
         self.status = status
         self.headers = headers
         self.body = body
+        self.cookies = cookies
+        self.storage = [:]
```

Comment:
I think it's cleaner to set all of the properties in the same place, the `init` method.

Only static properties need to be set at the class level.